pkg/scheduler/objects/node.go

```diff
@@ -312,6 +313,7 @@ func (sn *Node) RemoveAllocation(allocationKey string) *Allocation {
 		delete(sn.allocations, allocationKey)
 		sn.allocatedResource.SubFrom(alloc.GetAllocatedResource())
 		sn.availableResource.AddTo(alloc.GetAllocatedResource())
+		sn.allocatedResource.Prune()
```

Comment:
NIT should move this one line up for clarity

Reply:
Addressed in latest PR, thanks!